docker_builder/main.tf

```diff
@@ -0,0 +1,45 @@
+terraform {
+  required_providers {
+    null = "~> 2.1"
+    aws  = ">= 2.62"
+  }
+}
+
+module "git_branch" {
+  source  = "matti/resource/shell"
+  command = "git rev-parse --abbrev-ref HEAD"
+}
+
+module "git_sha" {
+  source  = "matti/resource/shell"
+  command = "git rev-parse HEAD"
+}
+
+locals {
+  registry   = "${var.repository}/${var.service}"
+  args       = join(" ", var.args)
+  git_branch = lower(replace(module.git_branch.stdout, "/", "-"))
+  git_sha    = module.git_sha.stdout
+}
+
+resource "null_resource" "build_image" {
+  triggers = {
+    image_tag = module.git_sha.stdout
+  }
+
+  provisioner "local-exec" {
+    command = <<EOT
+      eval $(aws ecr get-login --no-include-email);
```

Comment:
`aws ecr get-login` is old API
Can you update it to Use new `get-login-password` API

```shell
aws ecr get-login-password \
      --region ${var.region} \
      | docker login \
      --username AWS \
      --password-stdin ${aws_ecr_repository.image_repo.repository_url};
```